tests/v2/validation/pipeline/qase/reporter/main.go

```diff
@@ -194,19 +197,23 @@ func writeTestCaseToQase(client *qase.APIClient, testCase testcase.GoTestCase) (
 
 func updateTestInRun(client *qase.APIClient, testCase testcase.GoTestCase, qaseTestCaseID, testRunID int64) error {
 	status := fmt.Sprintf("%sed", testCase.Status)
-	elasedTime, err := strconv.ParseFloat(testCase.Elapsed, 64)
-	if err != nil {
-		return err
+	var elapsedTime float64
+	if testCase.Elapsed != "" {
```

Comment:
To understand this a bit better, when a test is skipped elapsed is an empty string and we want to make sure it is not skipped here right?